hydra-node/src/Hydra/Ledger/Cardano/Orphans.hs

```diff
@@ -0,0 +1,507 @@
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+-- | Orphans instances (mostly ToJSON/FromJSON) required by Hydra.Ledger.Cardano
+-- to satisfies our various internal interfaces.
+--
+-- Nothing to see here.
```

Comment:
So all of these are candidates we were considering to use the `cardano-api` ones? Especially this note from `Cardano.Api.Orphans` sounds like they intend to have JSON instances for their wrapper types we could rely on (in the mid-term): https://github.com/input-output-hk/cardano-node/blob/2cbe363874d0261bc62f52185cf23ed492cf4859/cardano-api/src/Cardano/Api/Orphans.hs#L66-L67

Reply:
Actually, this module only contains instances not present in Cardano.Api (that is, most `FromJSON` ones and some `ToJSON` ones)